readme.md

```diff
@@ -67,7 +67,7 @@ Force kill the process.
 Type: `boolean`<br>
 Default: `true`
 
-Kill all child processes along with the parent process. *(Windows only)*
+Kill all child processes along with the parent process.
```

Comment:
You have to update https://github.com/sindresorhus/fkill/blob/master/index.d.ts too.